src/ui/DidYouMean/DidYouMean.ts

```diff
@@ -142,7 +142,8 @@ export class DidYouMean extends Component {
       const correctedSentence = this.buildCorrectedSentence(results.queryCorrections[0]);
       this.correctedTerm = results.queryCorrections[0].correctedQuery;
 
-      const correctedWordEl = $$('a', {}, correctedSentence).el;
+      const correctedWordEl = $$('a', { href: 'javascript:void(0);' }, correctedSentence).el;
```

Comment:
For this one we could change it to a `button`, and adapt the styling so that it looks the same visually.

Reply:
And if you do that, then we don't need to set a tab index, or to block the "default" action with a void javascript function

Reply:
wouldn't that be breaking the dom in that case?

Reply:
Alright, I'll replace it with a button element. That should also simplify the tests.